tests/foreman/cli/test_syncplan.py

```diff
@@ -388,6 +386,8 @@ def test_positive_info_with_assigned_product(self):
             command
 
         :BZ: 1390545
+
+        :CaseImportance: Critical
```

Comment:
This test is integration level (as it touches two components - sync plans and products), isn't it? If so, it should be moved to tier2.

Reply:
@mirzal Your document on metadata is good, thanks for working on that.

Re integration implies tier 2, OK, changed new commit pushed.

Thank you